plugins/unpack/unzip_dir.sh

```diff
@@ -23,14 +23,13 @@ process_directory()
 			name=$(basename "${fn}")
 			process_directory "$1" "${fn}/" "$3${name}/"
 			last=$?
-			[ $last -gt 1 ] && ret=$last
+			[ $last -ge 1 ] && ret=$last
 		fi
 	done
 	return $ret
 }
 
 if [ "$6" != '' ] ; then
-	mkdir -p "$6"
```

Comment:
This mkdir wasn't necessary because it will do it in the process_directory function